src/Pages/MyUser/MyUser.jsx

```diff
@@ -22,7 +22,7 @@ export function MyUser() {
         };
         axios(config)
             .then((e) => {
-                e.data ? setImage(e.data) : setImage(image);
+                e.data ? setImage(e.data.data) : setImage(image);
```

Comment:
maybe we should use a name other than 'data', so that we don't have to use data.data. But approved for now